frontend/resourceadm/components/MergeConflictModal/MergeConflictModal.tsx

```diff
@@ -80,25 +68,25 @@ export const MergeConflictModal = ({
       ariaHideApp={false}
     >
       <h2 className={classes.modalHeader}>{t('merge_conflict.headline')}</h2>
+      <div className={classes.contentDivider} />
       <p className={classes.bodyText}>{t('merge_conflict.body1')} </p>
       <p className={classes.bodyText}>{t('merge_conflict.body2')}</p>
       <div className={classes.buttonWrapper}>
-        <Button
-          variant='quiet'
-          icon={<Download title='Download changed file' />}
-          iconPlacement='right'
-          onClick={handleDownloadChangedFilesZIP}
-        >
-          {'Last ned kun endret fil(er)'}
-        </Button>
-        <Button
-          variant='quiet'
-          icon={<Download title='Download whole repo' />}
-          iconPlacement='right'
-          onClick={handleDownloadRepoZIP}
-        >
-          {'Last ned hele repoet'}
-        </Button>
+        <div className={classes.downloadWrapper}>
+          <p className={classes.subHeader}>Last ned</p>
+          <Link
+            href={repoDownloadPath(org, repo)}
+            text='Last ned endret fil(er)'
```

Comment:
Kan ikke kodebasen så vet ikke om dere ha localizations på tekstene eller ikke?

Reply:
Det er planen i fremtiden. Er ikke lagt på noen sted for appen vår siden tekstene ikke er ferdigstilt / 100% enda.